src/main/flight/autopilot.c

```diff
@@ -105,3 +118,13 @@ float getAutopilotThrottle(void)
 {
     return throttleOut;
 }
+
+float getAutopilotHoverThrottleBatteryOffset(void) {
```

Comment:
Thanks for the hint about the battery compensation; I tried it, and it helps, but it doesn't handle having a different battery with a different voltage. I think this can be integrated later into the battery compensation feature since it is already there.

I will close this Br since it doesn't make sense to add this feature here.